cloudfront/wellcomelibrary.org/edge-lambda/package.json

```diff
@@ -7,15 +7,17 @@
   },
   "license": "MIT",
   "scripts": {
-    "test": "yarn jest",
-    "lint": "eslint src/**/*.ts --fix",
-    "build": "tsc -p tsconfig.build.json && yarn bundle",
+    "test": "yarn jest && yarn verifyStaticRedirects",
+    "lint": "eslint src/**/*.ts --fix && eslint *.ts --fix",
+    "build": "tsc -p tsconfig.build.json && yarn bundle ",
     "bundle": "cp package.json dist && cd dist && yarn install --production && zip -r wellcome_library_redirect.zip . && cd ..",
     "deploy": "yarn build && yarn test && node deploy",
     "dockerLoginLocal": "aws ecr get-login-password --region eu-west-1 --profile platform | docker login --username AWS --password-stdin 760097843905.dkr.ecr.eu-west-1.amazonaws.com",
     "dockerBuildLocal": "yarn dockerLoginLocal && docker build . -t weli_cf_lambdas",
     "dockerTestLocal": "yarn dockerBuildLocal && docker run weli_cf_lambdas yarn test",
-    "dockerDeployLocal": "yarn dockerTestLocal && docker run weli_cf_lambdas -v ~/.aws:/root/.aws yarn deploy"
+    "dockerDeployLocal": "yarn dockerTestLocal && docker run weli_cf_lambdas -v ~/.aws:/root/.aws yarn deploy",
+    "generateStaticRedirects": "ts-node generateStaticRedirects.ts",
```

Comment:
Could you stick this into the build/bundle command so it doesn't get forgotten?

Reply:
I deliberately did not do it as part of the build, and had the test step verify that it had been built. I can't remember why now, generating it in the build is better - i've done that.